fontforgeexe/cvpalettes.c

```diff
@@ -1558,6 +1559,50 @@ static BDFChar *BDFCharFromLayer(SplineChar *sc,int layer) {
 return( SplineCharAntiAlias(&dummy,ly_fore,24,4));
 }
 
+/**
+ *  \brief Recalculate the number of visible layers and reposition the scrollbar
+ *
+ *  \param [in] cv The charview
+ */
+static void CVLayers2Reflow(CharView *cv) {
+    extern int _GScrollBar_Width;
+    GGadget *scrollbar;
+    GRect cvl2size;
+
+    GDrawGetSize(cvlayers2, &cvl2size);
+    // Magic! I verified symmetry in Paint.NET
+    layer2.header_height = GGadgetGetY(GWidgetGetControl(cvlayers2, CID_VGrid)) - 1;
+    // Minus 2 because we always have the 'Guide' and 'Back' entries
```

Comment:
This is at odds with the comment in the struct for `visible_layers`, which states that only the guide layer is immune from scrolling.